src/templates/home/body/recentAnnotations/RecentAnnotations.js

```diff
@@ -45,8 +45,18 @@ const RecentAnnotations = () => {
                                     <Col md={{ span: 12 }}>
                                         <Row>
                                             {recentAnnotations.map((annotation, i) => {
-                                                if (!annotation['specimen']['specimenName']) {
-                                                    annotation['specimen']['specimenName'] = 'Undefined name';
+                                                /* Temporary solution */
```

Comment:
Create a list for the endpoints you would like. This seems something to be fixed in a endpoint where we can add a parameter to return only specimen annotations or only media annotations.

Reply:
Will create a list

Reply:
Get Annotation calls (all, recent, etc.): join op de target (Specimen/Media) om ipv een id de naam of iets anders van het target te tonen
Get Annotation calls (all, recent, etc.): paginate op calls, pageSize en pageNum parameters
Search Specimens call: total count van alle gevonden specimens
Get Media calls (all, recent): join op de target Specimen om ipv een id de naam te tonen

Reply:
Ok heb hier een user stories voor aangemaakt in Jira zodat we ze niet kwijt raken.
https://naturalis.atlassian.net/browse/DD-39